src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx

```diff
@@ -0,0 +1,192 @@
+
+For the `Sinatra/Rubytron/POST #purchaseCart` transaction there are many different areas where the transaction could be failing: databases, servers, controllers, middleware and more. You can toggle between each different segment of the transaction to identify trends and spikes.
+
+In this case, the controller, `Sinatra/Rubytron/POST` is the slowest segment and is taking 59.83% of the transactions total time. 
+
+    <img
+    title="Slowest transaction"
+    alt="A gif that demonstrates how to select the slowest transaction"
+    src={accountsSelectSlowestTransaction}
+/>
+
+</Step>
+
+<Step>
+### Take a look at throughput 
+
+While looking at response time for this transaction, you can also investigate throughput. Throughput is a way to measure the amount of work your application is handling. 
+
+Since you're experiencing a spike in errors in the `checkout-service` application and the `Sinatra/Rubytron/POST #purchaseCart` transaction, it's helpful to compare those errors with your throughput results. 
+
+If your throughput is very high during a time of service degradation it could indicate that your application is processing more work than it can handle. On the other hand, low throughput could indicate that your app isn't handling very many requests. This could simply mean that it's not used very much or it could mean that a service upstream that calls to your application is broken and requests aren't being processed. 
+
+<img
+    title="investigate throughput"
+    alt="A gif that shows you how to investigate throughput in the transaction details page"
+    src={apmInvestigateThroughputinAPM}
+/>
+
+</Step>
+
+<Step>
+
+### Investigate your transaction breakdown
+
+The transaction breakdown refers to the detailed breakdown of the time spent on each operation within a specific transaction. This breakdown helps developers understand the performance of the application and identify bottlenecks or areas for improvement.
+
+You can sort the transaction breakdown by **Category**, **% Time**, **Avg Calls/Txn**, and **Avg Time**. We recommend sorting by **% Time** to understand which piece of the transaction is taking up the most time within each transaction. 
+
+Let's examine the sections of the **Breakdown table** in the screenshot below:
+
+* **Category** refers to the type of call segment being monitored. 
+* **Controller** is a type of **category** that refers to model-view-controller architecture and is most likely a call made directly from your UI to your application. 
+* **External** is a type of **category** that represents a call being made that we began monitoring before we know where it's going.  
+*  **Middleware** is a type of **category** that most often refers to a dependency or library that is being used as part of your application's functionality. 
```

Comment:
```suggestion
* **Middleware** is a type of **category** that most often refers to a dependency or library that is being used as part of your application's functionality. 
```

Reply:
"that is being used as part of your application's functionality" sounds really vague and wonder if we can just say "refers to a dependency or library being used".